db/models/task.js

```diff
@@ -4,12 +4,20 @@ const { STRING } = require('sequelize');
 
 module.exports = db =>
   db.define('tasks', {
-    description: STRING,
-    status: STRING
+    description: {
+      type: STRING,
+      allowNull: false
+    },
+    status: { type: STRING
+      defaultValue: 'pending'
```

Comment:
Can you change this to 'Pending'? I think some stuff on the frontend checks for the capital P.